aws-cs-fargate/Infra/Program.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+using Pulumi;
+using Docker = Pulumi.Docker;
+using Ec2 = Pulumi.Aws.Ec2;
+using Ecs = Pulumi.Aws.Ecs;
+using Ecr = Pulumi.Aws.Ecr;
+using Elb = Pulumi.Aws.ElasticLoadBalancingV2;
+using Iam = Pulumi.Aws.Iam;
+
+class Program
+{
+    static Task<int> Main()
+    {
+        return Deployment.RunAsync(async () => {
```

Comment:
It's common to put `{` on a new line, also in our .NET code

Reply:
Alright. Not quite sure what the style is meant to be, e.g.

```
    LoadBalancers = {
        x
    },
```

or

```
    LoadBalancers =
    {
        x
    },
```

Do we have plans to add style/linter checking?

Reply:
The latter

Reply:
Oh my, the code is now twice as long! 🤦‍♂  But I shall resist the temptation to comment on style guidelines ... (oh wait, too late)